### PR TITLE
add guardduty dependency

### PR DESCRIPTION
**Pull request by samscottchef**
Signed-off-by: Sam Scott <sscott@chef.io>

uncommented the GuardDuty add dependency

## Description

I need the GuardDuty gem for adding a new resource in InSpec AWS

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New content (non-breaking change)
- [ ] Breaking change (a content change which would break existing functionality or processes)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: train-aws.gemspec [hunk 1]
```diff
@@ -117,7 +117,7 @@ Gem::Specification.new do |spec|
   # spec.add_dependency "aws-sdk-glue", "~> 1.0"
   # spec.add_dependency "aws-sdk-greengrass", "~> 1.0"
   # spec.add_dependency "aws-sdk-groundstation", "~> 1.0"
-  # spec.add_dependency "aws-sdk-guardduty", "~> 1.0"
+  spec.add_dependency "aws-sdk-guardduty", "~> 1.0"
```
Comment (rmoles) [inline train-aws.gemspec:120]: I see the newest version of guardduty is at `1.31`. Should we consider updating this to the latest?

Comment (samscottchef) [reply train-aws.gemspec:120]: Seems worthwhile to me

**Review by Schwad — COMMENTED** [review]
@samscottchef this looks good/reasonable enough to me 😄 

Whenever we add/bump dependencies I usually ping @inspec/foundation-team-reviewers for a day to ensure we're not missing any knowledge i.e. "we can't add this dependency because x"! - So unless they raise something by tomorrow happy to approve.

Do concur with @rmoles about using 1.31 if possible 👍

**Comment by samscottchef** [comment]
Latest commit updated to 1.31

**Review by tas50 — APPROVED** [review]
Works for me

**Review by Schwad — APPROVED** [review]
Thanks for this! Got foundation team's eyes on it and I'm happy to merge 🎉

**Comment by Schwad** [comment]
(and by "me" I mean someone who's actually authorized to merge on this PR 😂 , probably @rmoles )

**Review by skpaterson — APPROVED** [review]
Thanks @samscottchef !

**Pull request merged**